m2cgen/cli.py

```diff
@@ -109,7 +114,8 @@ def generate_code(args):
     sys.setrecursionlimit(args.recursion_limit)
 
     with args.infile as f:
-        model = pickle.load(f)
+        pickle_lib = __import__(args.lib)
```

Comment:
Unconditional loading of whatever module name is supplied into the CLI feels like a security concern to me. Can we perhaps maintain a list of supported libraries and check whether the supplied argument matches one of them?